Sources/Kitura/Router.swift

```diff
@@ -73,6 +73,22 @@ public class Router {
     /// and used to handle request's url parameters.
     fileprivate var parameterHandlers = [String : [RouterParameterHandler]]()
 
+    /// Contains the structures needed for swagger document generation
+    var swagger: SwaggerDocument
+
+    /// Computed property
+    ///
+    /// returns the current in-memory representation of the API as a json string.
```

Comment:
Returns the current in-memory representation of Codable routes as a Swagger document in JSON format, or nil if the document cannot be generated.

Reply:
done